taxonomy-editor-frontend/src/pages/editentry/ListTranslations.jsx

```diff
@@ -58,13 +65,23 @@ const ListTranslations = ({ nodeObject, setNodeObject }) => {
         setOpen(false);
     }
 
+    // Used for toggling visibility of a translation language
+    const handleToggleTranslationVisibility = (key) => {
+        const newShownTranslationLanguages = shownTranslationLanguages.includes(key) ? shownTranslationLanguages.filter(obj => !(key === obj)) : [...shownTranslationLanguages, key]
+        setShownTranslationLanguages(newShownTranslationLanguages);
+        newShownTranslationLanguages.sort()
+        localStorage.setItem('shownTranslationLanguages', JSON.stringify(newShownTranslationLanguages))
+        setOpen(false);
```

Comment:
nit: could we rename this `setOpen` to something more explicit (maybe `isOpenDialog`, `setIsOpenDialog` or something)